conference.js

```diff
@@ -71,7 +72,7 @@ const eventEmitter = new EventEmitter();
 let room;
 let connection;
 let localAudio, localVideo;
-let initialAudioMutedState = false, initialVideoMutedState = false;
+let initialAudioMutedState = false;
```

Comment:
Btw do you know if there's a way to fetch the current video muted status through the API ? Or are there events for that ? Otherwise with this toggle command you can get easily out of sync in case it fails to unmute